rollup.config.js

```diff
@@ -94,21 +120,49 @@ const primitivesConfig = Object.assign({}, configBase, {
     }),
   ],
   external: configBase.external.concat('react-primitives'),
+  plugins: configBase.plugins.concat(
+    replace({
+      __SERVER__: JSON.stringify(true),
+    })
+  ),
 })
 
 const noParserConfig = Object.assign({}, configBase, {
+  external: configBase.external.concat('stream'),
   input: 'src/no-parser/index.js',
   output: [
     { file: 'dist/styled-components-no-parser.es.js', format: 'es' },
     Object.assign({}, cjs, { file: 'dist/styled-components-no-parser.cjs.js' }),
   ],
+  plugins: configBase.plugins.concat(
+    replace({
+      __SERVER__: JSON.stringify(true),
```

Comment:
a little bit bummer that we have to put specify this in each config, but i guess there is not much that can be done about it, except wrapping config creation in a factory function, but im not sure if that would make things clearer, so 🤷‍♂️

Reply:
@Andarist Creating a config creation factory has really worked out well for me before. I'd love a refactor PR for this to make the config more readable again, if you don't mind? :)

Reply:
I won't be able to work on this any time soon, i have just too big OSS backlog at the moment :p but anyway it should be done as a followup PR to this and probably also not the biggest priority. 

That being said I will work on [microbundle](https://github.com/developit/microbundle) and would like to hide most of the stuff that was made here in it. However `microbundle` is "0" configuration tool at the moment and it's good to have a control and possibility to extend a config, so I'm not sure if it will be a fit for you yet.